open_academy/views/open_academy_session_views.xml

```diff
@@ -0,0 +1,48 @@
+<?xml version='1.0' encoding='utf-8'?>
+<odoo>
+    <!-- Action of the view configuration -->
+    <record model="ir.actions.act_window" id="session_action_list">
+        <field name="name">session.action.list</field>
```

Comment:
Sorry, I got confused with this. I thought it was the view and it is the action. This is what the user sees when he opens the view, so please change the name.

```suggestion
        <field name="name">Sessions</field>
```
Sorry 🙊